manim/mobject/types/point_cloud_mobject.py

```diff
@@ -91,8 +95,9 @@ def add_points(self, points, rgbas=None, color=None, alpha=1):
     def set_color(self, color=None, family=True):
         if config["renderer"] == "opengl":
             color = self.color if not color else color
-            opacity = 1 if not self.opacity else self.opacity
-            return super().set_color(color=color, opacity=opacity)
+            # temp fix due to clash between opengl and cairo implementations
+            # as opengl will pass opacity and not family for set_color
+            return super().set_color(color=color, opacity=family)
```

Comment:
This was more of a hack, it is always set as a float as the only method calling it defaults it to 1, but this one example of why it might be better to separate the opengl / cairo implementations because parts don't fit together like this method.